src/style.css

```diff
@@ -0,0 +1,42 @@
+body {
+  margin: 0;
+
+}
+
+.stars {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: row;
+}
+
+.stars > * {
+  background-image: url(images/star.svg);
+  height: 16px;
+  width: 16px;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars__star:nth-child(n+2) {
```

Comment:
`:not(:first-child)` is more clear selector here
I needed some time to understand your one